docs: readability edit of the w3-account spec (#15)
web3-storage/specs: w3-account.md

We also propose an account authorization flow that would allow an authorized agent to act on behalf of the account [principal][].

The sequence diagram below gives an overview of a complete authorization flow as described in this document.

```mermaid
sequenceDiagram
  participant Agent as 👩‍💻<br/><br/>did:key:zAgent
  participant W3 as 🌐<br/><br/>did:dns:web3.storage #32;
  participant Email as 📬<br/><br/>alice@web.mail

  Agent ->> W3: ucan/issue
  Note right of Agent:🎟<br/>with: did:key:zAgent<br/>as: did:mailto:alice@web.mail
  W3 ->> Email: ✉️ Verification email
  Email ->> W3: 🔗 Approve
  W3 -->> Agent: ucan/sign
    Note right of Agent:🎫<br/>with: did:dns:web3.storage<br/>as: did:key:zAgent
```

# Terminology

## Account

An account is a [principal][] identified by [`did:mailto`][] identifier.

> It MUST be a DID identifier as opposed to `mailto:` URI to be a valid principal in the UCAN protocol.

## Authorization

> A User MAY authorize an agent to represent their account by delegating capabilities to it. 

Reviewer: ```suggestion
A user MAY authorize an agent to represent their account by delegating capabilities to it. 
```

MUST be a [`did:key`][] of the principal which MAY sign [UCAN][]s issued by an account principal in [`aud`](#sign-aud).

## Delegate Access

When an agent creates a new space, it MAY delegate all capabilities or a subset, to a desired account. This would allow a user to reclaim the delegated capabilities via a different agent.

> > a delegation of capabilities to `did:key:zAliceSpace` space to the user account `alice@web.mail`

Reviewer: ```suggestion
> A delegation of capabilities to `did:key:zAliceSpace` space to the user account `alice@web.mail`
```

# Limitations

Using delegation from specific authority as an authorization proof limits it to the contexts in which signing authority is trusted. It is a reasonable compromise when receiver and the issuer of the proof are the same authority (as is the case for a service like web3.storage).

We will remove this limitation in the future the account system by upgrading [authorization][] to use the [UCAN mailto][] specification, so that email ownership may be verified without the [email verification][] step described above, allowing this accounts system to be used from in a global, permissionless context.

# Related Notes

## Free provider

> web3.storage offers one "free provider" per account. It will be denied if a `consumer` space is not specified or has already been applied.

Reviewer: ```suggestion
web3.storage offers one "free provider" per account. It will be denied if a `consumer` space is not specified or if account has already claimed it with a different `consumer` space.
```
